src/commcare_cloud/manage_commcare_cloud/install.py

```diff
@@ -23,13 +23,20 @@ def run(self, args, unknown_args):
         put_virtualenv_bin_on_the_path()
         if not os.path.exists(ANSIBLE_ROLES_PATH):
             os.makedirs(ANSIBLE_ROLES_PATH)
+        
+        if not os.path.exists(ANSIBLE_COLLECTION_PATHS):
+            os.makedirs(ANSIBLE_COLLECTION_PATHS)
 
         env['ANSIBLE_ROLES_PATH'] = ANSIBLE_ROLES_PATH
-        cmd_parts = ['ansible-galaxy', 'install', '-f', '-r', os.path.join(ANSIBLE_DIR, 'requirements.yml')]
-        cmd = ' '.join(shlex_quote(arg) for arg in cmd_parts)
-        print_command(cmd)
-        p = subprocess.Popen(cmd, stdin=subprocess.PIPE, shell=True, env=env)
-        p.communicate()
+        env['ANSIBLE_COLLECTION_PATHS'] = ANSIBLE_COLLECTION_PATHS
+        cmd_roles_parts = ['ansible-galaxy', 'install', '-f', '-r', os.path.join(ANSIBLE_DIR, 'requirements.yml')]
```

Comment:
Just noticed this. I think you have to specify `role` here to install the roles:
```suggestion
        cmd_roles_parts = ['ansible-galaxy', 'role', 'install', '-f', '-r', os.path.join(ANSIBLE_DIR, 'requirements.yml')]
```

From the docs:
> While both roles and collections can be specified in one requirements file, they need to be installed separately. The ansible-galaxy role install -r requirements.yml will only install roles and ansible-galaxy collection install -r requirements.yml -p ./ will only install collections.